.github/workflows/build-binaries.yml

```diff
@@ -71,11 +71,17 @@ jobs:
             cross: true
           
           # macOS
-          - build: macos
+          - build: macos-x86_64
             os: macos-latest
             rust: stable
             target: x86_64-apple-darwin
             cross: false
+            
+          - build: macos-aarch64
```

Comment:
Whats that?

Reply:
The builder for MacOS ARM, copied from the same place as the other ones: https://github.com/crodjer/sysit/blob/main/.github/workflows/release.yml#L67-L71